### DISCUSSION TITLE
single-phase benchmark validation-getting high value

### DISCUSSION THREAD

**Discussion by rksin8** [idaholab/moose#19665] [Q&A Modules: General] [answered]
Hi,
I am working on a single-phase hydromechanical benchmark problem:
![image](https://user-images.githubusercontent.com/22324848/146664920-50728d1d-e6ed-46e9-8eb4-78eb0ae7d82a.png)
Injection rate: 0.02 kg/s/m

I am getting the same shape for pressure but high values:
![image](https://user-images.githubusercontent.com/22324848/146673412-b52cf378-7464-475a-8a77-168a3ad9ef65.png)

Displacement:  
![image](https://user-images.githubusercontent.com/22324848/146673421-43f00729-2c1e-4281-8d0f-bf19a39115c7.png)

My input file & grid file:  https://github.com/rksin8/benchmark1p. 

Any idea to fix the problem would be greatly appreciated!

Thank you

**Comment by cpgr** [comment, marked as answer]
At a guess, it is probably the non-standard units not being consistent throughout the input file somewhere. The `PorousFlowFullySaturated` action takes a `pressure_unit` and `time_unit` input parameter to make sure that the correct fluid properties are computed, see https://mooseframework.inl.gov/source/actions/PorousFlowFullySaturated.html

Does simply adding `pressure_unit = MPa` and `time_unit = days` to this action fix it?
